README.md

```diff
@@ -597,12 +597,12 @@
     });
 
     // good
-    [1, 2, 3].map((x) => {
-      return x * x;
-    });
+    [1, 2, 3].map(x => x * x);
```

Comment:
Good point.

>  let's update the example function to be more than one-line

I guess that’d be better – then all our examples will follow all rules :)